doc/man3/SSL_shutdown.pod

```diff
@@ -54,10 +54,7 @@ When the application is the first party to send the "close notify"
 alert, SSL_shutdown() will only send the alert and then set the
 SSL_SENT_SHUTDOWN flag (so that the session is considered good and will
 be kept in the cache).
-SSL_shutdown() will then return 0.
-
-SSL_shutdown() can return -1, in which case SSL_shutdown() should be called
-again.
+If succesful, SSL_shutdown() will return 0.
```

Comment:
succesful -> successfull

Reply:
If you want to correct it, do it properly. :)